README.md

```diff
@@ -1,36 +1,28 @@
-# Check Digit < Insert name > Library
+# Memoize
 
-The Check Digit < Insert Name > library is a library for Check Digit services to deal with [ short description about what the library does]
+Copyright (c) 2023 [Check Digit, LLC](https://checkdigit.com)
 
-Features:
+Memoize is a small async function used by Check Digit services for memoizing promises with cache eviction on reject.
+The promise being cached guarantees the underlying function is called at most once for a given set of arguments, unless
+the promise is rejected. If the promise is rejected, it will be removed from the memoize cache.  
+This allows the function to be retried.
 
-- Feature 1
-- Feature 2
-- Feature3
-- (...)
-
-### Installing
-
-`npm install @checkdigit/< Insert name >`
-
-### Use
-
-It could be. i.e.
+### Install
 
 ```
-import * as insertName from '@checkdigit/< Insert name >';
+$ npm install @checkdigit/memoize
+```
 
-const someVariable = insertName();
+### Use
 
 ```
+import { memoize } from '@checkdigit/memoize';
```

Comment:
I don't think this code would work, there's only a default import.   believe should be `import memoize from '@checkdigit/memoize';`